src/utils.c

```diff
@@ -435,6 +435,87 @@ SEXP startsWithAny(const SEXP x, const SEXP y, SEXP start) {
   return ScalarLogical(false);
 }
 
+// if (length(x)) length(x[[1L]]) else 0L
+int NROW(SEXP x) {
```

Comment:
if these are only useful for mergelist.c, shouldn't we keep `NROW`/`NCOL` defined there instead?

I also worry `NROW`/`NCOL` are confusing because they conflict a bit with how the R-level functions with the same name operate.

Reply:
I see it's used by `setDT()` though it's commented out here. I will try renaming.